conda_smithy/feedstock_content/.gitattributes

```
@@ -9,6 +9,7 @@ bld.bat text eol=crlf
 # github helper pieces to make some files not show up in diffs automatically
 .azure-pipelines/* linguist-generated=true
 .circleci/* linguist-generated=true
+.ci_support/README.md linguist-generated=true
```

Currently it is [plain text without an extension]( https://github.com/conda-forge/conda-smithy/blob/24598d9fe5f131b7505ec64b40ecaf13678ff461/conda_smithy/configure_feedstock.py#L2256 )

```suggestion
.ci_support/README linguist-generated=true
```

That said, supportive of making it Markdown instead if preferred